### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
[DEMO LINK](https://SerhiiUnhurian.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/components/PeopleTable.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+          <td>Jacobus van Brussel</td>
+        </tr>
+
+        <tr data-cy="person">
+          <td>
+            <a href="#/people/philibert-haverbeke-1907">
+              Philibert Haverbeke
+            </a>
+          </td>
+
```
Comment (etojeDenys) [inline src/components/PeopleTable.tsx:57]: remove all comments

File: src/components/Person.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+type Props = {
+  person: PersonType;
+};
+
+export const Person: React.FC<Props> = ({ person }) => {
+  const { born, died, sex, motherName, fatherName, mother, father, slug } = person;
+
+  const { personSlug } = useParams();
+
+  console.log(person);
```
Comment (etojeDenys) [inline src/components/Person.tsx:16]: image

File: src/components/PersonLink.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import classNames from 'classnames';
+
+type Props = {
+  person: PersonType;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, slug } = person;
+
+  console.log(person);
```
Comment (etojeDenys) [inline src/components/PersonLink.tsx:13]: image

File: src/components/PeopleTable.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
```
Comment (etojeDenys) [inline src/components/PeopleTable.tsx:27]: image
fix all those errors

**Review by etojeDenys — CHANGES_REQUESTED** [review]
fix the demo link too

File: src/components/PeopleTable.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+              pers => pers.name === person.motherName,
+            );
+          }
+
+          if (person.fatherName) {
+            person.father = people.find(
+              pers => pers.name === person.fatherName,
+            );
+          }
+
```
Comment (IvanFesenko) [inline src/components/PeopleTable.tsx:40]: Do not mutate original array

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check my comment

**Review by etojeDenys — APPROVED** [review]
awesome

**Pull request closed without merging**